http/e2e/cmd/httpe2e/main.go

```diff
@@ -19,24 +19,7 @@ import (
 // --proxy-hostport: Proxy endpoint used to perform requests. Defaults to "localhost:8080".
 // --httpbin-hostport: Upstream httpbin endpoint, used for health checking reasons. Defaults to "localhost:8081".
 
-// Expected Coraza configs:
-/*
-SecRuleEngine On
-SecRequestBodyAccess On
-SecResponseBodyAccess On
-SecResponseBodyMimeType application/json
-# Custom rule for Coraza config check (ensuring that these configs are used)
-SecRule &REQUEST_HEADERS:coraza-e2e "@eq 0" "id:100,phase:1,deny,status:424,log,msg:'Coraza E2E - Missing header'"
-# Custom rules for e2e testing
-SecRule REQUEST_URI "@streq /admin" "id:101,phase:1,t:lowercase,log,deny"
-SecRule REQUEST_BODY "@rx maliciouspayload" "id:102,phase:2,t:lowercase,log,deny"
-SecRule RESPONSE_HEADERS:pass "@rx leak" "id:103,phase:3,t:lowercase,log,deny"
-SecRule RESPONSE_BODY "@contains responsebodycode" "id:104,phase:4,t:lowercase,log,deny"
-# Custom rules mimicking the following CRS rules: 941100, 942100, 913100
-SecRule ARGS_NAMES|ARGS "@detectXSS" "id:9411,phase:2,t:none,t:utf8toUnicode,t:urlDecodeUni,t:htmlEntityDecode,t:jsDecode,t:cssDecode,t:removeNulls,log,deny"
-SecRule ARGS_NAMES|ARGS "@detectSQLi" "id:9421,phase:2,t:none,t:utf8toUnicode,t:urlDecodeUni,t:removeNulls,multiMatch,log,deny"
-SecRule REQUEST_HEADERS:User-Agent "@pm grabber masscan" "id:9131,phase:1,t:none,log,deny"
-*/
+// Expected Coraza config see http/e2e.go
```

Comment:
I would be a bit more verbose considering that loading these directives is a very needed step to run e2e and we need to make they actually take a look at the file we are pointing them 
```suggestion
// A dedicated set of directives is expected to be loaded for e2e testing. Refer to the `Directives` const in http/e2e.go.
```

Reply:
Done Thanks 😄